tests/unit/package_managers/test_gomod.py

```diff
@@ -49,11 +50,21 @@ def setup_module():
 
 
 @pytest.fixture
-def gomod_request(tmp_path: Path) -> Request:
+def gomod_input_packages() -> list[dict[str:str]]:
```

Comment:
should be `dict[str, str]`

```
x.py:1: error: "dict" expects 2 type arguments, but 1 given
x.py:1: error: Invalid type comment or annotation
x.py:1: note: did you mean to use ',' instead of ':' ?
Found 2 errors in 1 file (checked 1 source file)
```